Sming/Libraries/Adafruit_ILI9341/Adafruit_ILI9341.cpp

```diff
@@ -157,7 +157,8 @@ void Adafruit_ILI9341::begin(void) {
 	transmitCmdData(0xE1, data, 15);    	//Set Gamma
 
 	transmitCmd(0x11);    	//Exit Sleep
-	os_delay_us(120000);
```

Comment:
it is mandatory change. without this this library do not compile any more. Old os_delay_us declaration was uint32_t and new, from esp sdk 2.1.0 is uint16_t, so I made such "strange" change here.